src/Tribe/JSON_LD/Organizer.php

```diff
@@ -59,4 +59,21 @@ public function get_data( $post = null, $args = array( 'context' => false ) ) {
 		return array( $post_id => $data );
 	}
 
+	/**
+	 * Get a link to the event
+	 *
+	 * @param  int|WP_Post  $post The Post Object or ID
+	 *
+	 * @return false|string Link to the event or false
+	 */
+	protected function get_link( $post ) {
+		if ( class_exists( 'Tribe__Events__Pro__Main' ) ) {
```

Comment:
I completely agree. However, this MR does not affect Pro. Further, we need to make some overhauls to all three of these functions, which is ticketed: https://central.tri.be/issues/33734

Perhaps we can just add this item to that overhaul? Would that work @bordoni ?